.circleci/config.yml

```diff
@@ -22,15 +22,15 @@ jobs:
             - .git/lfs
        - restore_cache:
            keys:
-             - v7-import-{{ .Branch }}
-             - v7-import-master
+             - v8-import-{{ .Branch }}
+             - v8-import-master
```

Comment:
huh, have I missed something? What is this change?

Reply:
I changed it as the import step in CI is starting to take a while, so this refreshes the caches.